finetuner/client/client.py

```diff
@@ -227,14 +225,19 @@ def _push_data_to_hubble(
 
     def download_model(
         self, experiment_name: str, run_name: str, path: str = FINETUNED_MODELS_DIR
-    ) -> requests.Response:
-        """Download finetuned model from Hubble by its ID.
+    ) -> List[str]:
+        """Download finetuned model(s) from Hubble by its ID.
 
         :param experiment_name: The name of the experiment.
         :param run_name: The name of the run.
         :param path: Directory where the model will be stored.
-        :returns: A str object indicates the download path on localhost.
+        :returns: A list of str object(s) that indicate the download path on localhost.
```

Comment:
localhost? You mean local disk?

Reply:
I honestly just copied that from hubble's `download_artifact` docstring. But I see your point, it's a bit confusing here.